APIWrapper/SchoolScheduleProxy.cs

```diff
@@ -29,9 +29,11 @@ public IEnumerable<ScheduleHourContent> GetByRoomNumber(string roomNumber)
             return CallScheduleContentApi(3, roomNumber);
         }
 
-        public IEnumerable<ScheduleHourContent> GetBySubjectCode(string subjectCode)
+        public IEnumerable<ScheduleHourContent> GetBySubjectCode(string subjectCode, string yearOfStudy, string studyType)
         {
-            string addition = ",5,1,1" + GetCurrentSemesterShortCut();
+            char numStudyType = studyType.Contains("bak") ? '2' : studyType.Contains("inž") ? '1' : studyType.Contains("dokt") ? '3' : ' ';
```

Comment:
Maybe add "bak" ... as constants in code. Or enums with function that returns number. But this is optional.